### PR TITLE
+act #18487 Support custom supervision strategy in BackoffSupervisor

### PR DESCRIPTION
**Pull request by patriknw**
- and forward messages from child to parent
  (needed for sharding passivation)

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #18487

File: akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala [hunk 1]
```diff
@@ -156,6 +210,9 @@ final class BackoffSupervisor(
     case GetCurrentChild ⇒
       sender() ! CurrentChild(child)
 
+    case msg if child.contains(sender()) ⇒
+      context.parent.forward(msg)
```
Comment (rkuhn) [inline akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala]: Hmm, this would need to be mentioned in the release notes somewhere.

**Comment by rkuhn** [comment]
LGTM

**Comment by johanandren** [comment]
LGTM

**Pull request merged**